doc/TROUBLESHOOTING.md

```diff
@@ -39,7 +39,7 @@ error: could not compile `aardwolf` (bin "aardwolf-server" test) due to previous
 **Resolution:**
 The key here after the wall of text --> `note: ld: library not found for -lpq`. The `lpq` package is normally installed as part of PostgreSQL (Lib PQ)
 
-Debian/Ubunutu: `apt install libpq`
+Debian/Ubunutu: `apt install libpq-dev`
```

Comment:
**Suggestion:** There's a typo in the word 'Ubuntu'. It should be corrected to maintain the professionalism and accuracy of the documentation.
```suggestion
Debian/Ubuntu: `apt install libpq-dev`
```